drivers/platform/aducm3029/uart.c

```diff
@@ -45,6 +45,10 @@
 #include "uart_extra.h"
 #include <stdlib.h>
 
+#ifndef min
+#define min(x, y) (((x) > (y)) ? (y) : (x))
```

Comment:
min() is defined in util.h. We should include this header instead of redefining it.

Reply:
I have some conflicts with util.h. I will solve the problem in an other way if defining min is not ok

Reply:
Let's fix util.h then.

Reply:
I solved by including util.h after <stdlib.h>